app/js/lib/form_components.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+var objectAssign = require( 'object-assign' ),
+	actions = require( './actions' ),
+	createDefaultChangeHandler = function ( store, contentName ) {
+		return function ( evt ) {
+			store.dispatch( actions.newChangeContentAction( contentName, evt.target.value ) );
+		};
+	},
+	RadioComponent = {
+		element: null,
+		contentName: '',
+		onChange: null,
+		render: function ( formContent ) {
```

Comment:
After reading the code calling this I realized this gets the whole form content... which the var name kinda suggests :) That still makes the method name odd though, as this component does not render the whole form content.